contents/handbook/company/team/paolo-damico.md

```diff
@@ -4,8 +4,26 @@ sidebar: Handbook
 showTitle: true
 ---
 
+## Q1 2022 Goals
```

Comment:
Would love to see something specific about quality in both strategy and execution